app/Repositories/BatchRepository.php

```diff
@@ -58,14 +58,14 @@ public function addDayMonth($batch)
                 'year' => $year
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Repositories/BatchRepository.php</br>**Line**: 58</br>Each line in an array declaration must end in a comma